packages/react-transport-dom-webpack/src/ReactFlightWebpackPlugin.js

```diff
@@ -8,17 +8,170 @@
  */
 
 import {mkdirSync, writeFileSync} from 'fs';
-import {dirname, resolve} from 'path';
+import {dirname, resolve, join} from 'path';
 import {pathToFileURL} from 'url';
 
+// This can't be loaded as an ESM module.
```

Comment:
I guess this is probably getting inlined. I should treat it as external.

Reply:
What is this? Does webpack use it?

Reply:
yea

Reply:
Should probably be a peer?

Reply:
I guess exposing it to the user is also a bit weird.

Reply:
It’s just a dependency. It has no shared state. I only use it so it has a chance to dedupe and minimize dependencies/differences.

Reply:
Should add it to `package.json` then? So Yarn PnP can work etc.